language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/sourceprune/LHSCompletionsTokenTraverser.java

```diff
@@ -32,7 +35,7 @@
  * 
  * @since 0.995.0
  */
-class LHSTokenTraverser extends AbstractTokenTraverser {
+public class LHSCompletionsTokenTraverser extends AbstractTokenTraverser {
```

Comment:
Is it possible to make the class package-private since we do not expose this as a public API?